dune/grid/cpgrid/Entity.hpp

```diff
@@ -331,14 +348,14 @@ namespace Dune
 //          }
 
             /// Const member by pointer operator.
-            Entity* operator->() const
+            const Entity* operator->() const
             {
                 assert(Entity::valid());
                 return const_cast<EntityPointer*>(this); // const_cast-hack added because of error in vtkwriter.hh
```

Comment:
I thought about it. Maybe I should not change the signature just before the release. 
However, CpGrid is quite out of date when it comes to recent dune interface changes. 
For dune 2.4 we can return an Entity here. I think the operator-> has vanish.